pallets/nft/src/lib.rs

```diff
@@ -31,6 +31,9 @@ pub mod benchmarking;
 
 decl_storage!(
     trait Store for Module<T: Config> as NFT {
+        /// The total number of NFTs per identity.
+        pub BalanceOf get(fn balance_of): double_map hasher(blake2_128_concat) Ticker, hasher(identity) IdentityId => Balance;
```

Comment:
We should use a different name here since NFTs don't really have balances.

Reply:
Also I wonder if we need to use a `u128` (Balance) for tracking the number of NFTs an identity has.  The `NFTId` type is a `u64`, so there can't be more then `u64::MAX` minted NFTs for each collection (`Ticker`).  Maybe define a `NFTCount` type that also wraps `u64`.

No need to use ONE_UNIT, since NFT is not divisible.